scripts/create_mac_app_final.py

```
@@ -0,0 +1,268 @@
+echo ""
+echo "To run the app:"
+echo "1. Open Finder"
+echo "2. Go to Applications"
+echo "3. Double-click 'iMessage Exporter'"
+echo ""
+echo "Or run from Terminal:"
+echo "open ~/Applications/iMessage\\ Exporter.app"
+echo ""
+echo "Note: You may need to allow the app in System Preferences > Security & Privacy"
+echo "The app will request access to your Messages database when you first run it."
+'''
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)